scripts/ci-scripts/react-tests.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
```

Comment:
This file can probably be removed.  The legacy react native tests have been disabled and will be replaced by the integration tests.